NuCivic/dkan
# Show other value on block license dataset page #CIVIC-4488

Issue: https://jira.govdelivery.com/browse/CIVIC-4488

## Description

If you select other value for license field this value isn't showed on the dataset page (Block left). 



## How to reproduce

1.  Create or edit a dataset and select "Other" as the license value. Specify a value on the field that appears.
2. Save the dataset.
3. Go to the dataset page and confirm that "License not specified" is shown on the license block while the license is being displayed correctly on the information section.

## QA Tests

- [x] Add other value to license field and go to dataset page. You should see the value you inserted on license field.
